sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -82,6 +82,11 @@ function ListingsSearchCombined(props: ListingsSearchCombinedProps) {
 
     searchParams.current = params
 
+    document.getElementById("listings-outer-container")?.scrollTo(0, 0)
+    document.getElementById("listings-list")?.scrollTo(0, 0)
+    document.getElementById("listings-list-expanded")?.scrollTo(0, 0)
+    window.scrollTo(0, 0)
```

Comment:
I know this isn't part of your PR, but do you mind removing the console log right below this. I think that was accidentally left in during development